addons/number-pad/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Always show number pad",
+  "description": "Show Scratch's number pad input when editing number fields on all devices instead of only touchscreen devices.",
```

Comment:
I'm not sure if users are aware of the number pad. So I don't think it makes sense to use words such as "always".
It may make more sense to say things such as that the number pad input appears below the block, and that the idea is to remove the need to use the keyboard (either physical or virtual)

Reply:
Apparently, laptops with touch screen already support this feature (vanilla Scratch), so this addon would only benefit users that want to use a physical mouse to enter the numbers - and users without a physical keyboard would have no reason to enable this addon.

Reply:
> Apparently, laptops with touch screen already support this feature (vanilla Scratch), so this addon would only benefit users that want to use a physical mouse to enter the numbers - and users without a physical keyboard would have no reason to enable this addon.

That's the entire point of the number pad– to show on touchscreen devices.